src/odemis/acq/millmng.py

```diff
@@ -355,7 +355,7 @@ def _mill_all_settings(self, site: CryoFeature):
 
                         self._scanner.probeCurrent.value = milling_settings.current.value
                         self._future.running_subf = mill_rectangle(milling_settings.roi.value, self._scanner,
-                                                                   iteration=self._iterations[setting_nb],
```

Comment:
This will also provide iteration 1 as it is calculated from the init of this class.

Reply:
No. This is the case where there is a dritf correction. It's used line 351 for the `for` loop. It's not 1.